parachain/primitives/router/src/outbound/mod.rs

```diff
@@ -134,11 +135,11 @@ where
 		// convert fee to MultiAsset
 		let fee = MultiAsset::from((MultiLocation::parent(), fee)).into();
 
-		Ok((ticket.encode(), fee))
+		Ok(((ticket.encode(), XcmHash::default()), fee))
```

Comment:
Should be `message_id` for the ticket not the `XcmHash::default()`.

Reply:
I think we can fix this in a follow up PR, as we need to make some other related changes to message IDs anyway.